xlators/storage/posix/src/posix.h

```diff
@@ -202,6 +202,8 @@ struct posix_private {
     pthread_t health_check;
 
     double disk_reserve;
+    /* seconds for check disk reversion  */
+    uint32_t disk_reserve_check_interval;
```

Comment:
I think you should declare disk_reserve_check_interval in under glusterfs_ctx_t because currently we have single posixreserve thread for all the bricks while brick_mux is enabled.If user changes the value for any volume it should reset ctx->disk_reserve_check_interval variable and use the same at the time of using the interval
in posix_ctx_disk_thread_proc.

Reply:
@mohit84  ok,i agree.